### DISCUSSION TITLE
Add a new disk to the existing volume server, how to rebalance files to all disks again?

### DISCUSSION THREAD

**Discussion by meotimdihia** [seaweedfs/seaweedfs#5880] [Q&A] [answered]
We had this feature. I tried, but it doesn't work:

```
/dev/sda1      21483233280 20899490828   583742452  98% /mnt/disk1
/dev/sdb1      21483233280   411383748 21071849532   2% /mnt/disk2
..........
```

The volume command: ` weed volume -port 8080 -max 0,0,0,0,0,0,0,0  -dir /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4,/mnt/disk5,/mnt/disk6,/mnt/disk7,/mnt/disk8 ......`

**Comment by chrislusf** [comment, marked as answer]
the feature is not implemented. new volumes will go to disks with free spaces.
